src/System.Diagnostics.Process/src/System/Diagnostics/ProcessThread.OSX.cs

```diff
@@ -13,8 +13,11 @@ public partial class ProcessThread
         /// </summary>
         private ThreadPriorityLevel PriorityLevelCore
         {
-            get { throw new PlatformNotSupportedException(); }
-            set { throw new PlatformNotSupportedException(); }
+            // We couldn't find a POSIX API on OSX
```

Comment:
I guess I mean, there is no POSIX API so platform specific was used and it does not exist on OSX. Right @Priya91 ?